source/contact.blade.php

```diff
@@ -8,66 +8,119 @@
+            ></textarea>
         </div>
-
-        <div class="w-full px-3 md:w-1/2">
-            <label class="block text-gray-800 text-sm font-semibold mb-2" for="contact-email">
-                Email Address
-            </label>
-
+    
+        <div class="flex justify-end w-full">
             <input
-                type="email"
-                id="contact-email"
-                placeholder="email@domain.com"
-                name="email"
-                class="block w-full border shadow rounded-lg outline-none mb-2 px-4 py-3"
-                required
+                type="submit"
+                value="Submit"
+                class="block bg-blue-500 hover:bg-blue-600 text-white text-sm font-semibold leading-snug tracking-wide uppercase shadow rounded-lg cursor-pointer px-6 py-3"
             >
         </div>
-    </div>
+    </form>
+    <div>
+        <div>
+            <h3>What is next?</h3>
+        </div>
+        <div>
+            <i class="fa fa-arrow-right" aria-hidden="true"></i>
+            <span>  
+                We’ll contact you with more information on the next steps within the next 2 - 3 business
+                days. 
+            </span>
+        </div>
 
-    <div class="w-full mb-12">
-        <label class="block text-gray-800 text-sm font-semibold mb-2" for="contact-message">
-            Message
-        </label>
+        <div>
+            <span><i class="fa fa-arrow-right" aria-hidden="true"></i> </span>
+                <span>  
+                    We would also love to collect all your project requirements, get clarifications on your
+                    business objectives, and expectations towards your project.         </span>
+        </div>
 
-        <textarea
-            id="contact-message"
-            rows="4"
-            name="message"
-            class="block w-full border shadow rounded-lg outline-none appearance-none mb-2 px-4 py-3"
-            placeholder="A lovely message here."
-            required
-        ></textarea>
+        <div>
+            <span><i class="fa fa-arrow-right" aria-hidden="true"></i> </span>
+                <span>  
+                    Thereafter, we set to work.         
+                </span>
+        </div>
+        <br>
+        <br>
     </div>
-
-    <div class="flex justify-end w-full">
-        <input
-            type="submit"
-            value="Submit"
-            class="block bg-blue-500 hover:bg-blue-600 text-white text-sm font-semibold leading-snug tracking-wide uppercase shadow rounded-lg cursor-pointer px-6 py-3"
-        >
+        <div class="mb-8">
+            <div>
+            <h4>
+                You can reach us on ...
+            </h4>
+            </div>
+            <div>
+                <span class="icon">        
+                    <i class="fa fa-phone-square" aria-hidden="true"></i>
+                </span>
+                <span>  phone number:  </span>
+                <span>   +233-26-123-456  </span>
```

Comment:
Let's include the number from the configure file. This way the update is done once and it reflects on all pages

`{{ $page->contactNumber }}`

Look at how I did it on the staging branch of this repository.

I think you worked on the master branch

Reply:
okay